apps/bfd-server/dev/devenv-readme.md

```diff
@@ -36,29 +36,42 @@ For your convenience, a dev-only-really-don't-use-these-anywhere-else server key
     ```
     $ keytool -genkeypair -alias server -keyalg RSA -keysize 4096 -dname "cn=localhost" -ext "san=dns:localhost,ip:127.0.0.1" -validity 3650 -keypass changeit -keystore bluebutton-server.git/dev/ssl-stores/server-keystore.jks -storepass changeit
     ```
+    1. Do note that _all_ JKS keystores have since been updated to use the more widely accepted standard PKCS12 format. Using `server-keystore.jks` as an example, this conversion was achieved with the following `keytool` command:
+
+        ```
+        keytool -importkeystore -srckeystore ./server-keystore.jks -destkeystore server-keystore.pfx -srcstoretype jks -deststoretype pkcs12 -srcstorepass changeit -deststorepass changeit -srcalias server -destalias server -srckeypass changeit -destkeypass changeit -noprompt
+        ```
+    2. Do note that this document will continue to refer to the (historical) certificate as if it were in JKS format and named `server-keystore.jks`. Any reference to `server-keystore.jks` can be replaced with `server-keystore.pfx`
     
```

Comment:
Any reason not to just find and replace the instances here?

Reply:
> Any reason not to just find and replace the instances here?

Between Brian, Mitch, and I, we were looking at this very thing.

I realize that the commit history does a fine job by itself recording things for posterity, but with some of the shifts recently and work we're scheduling in the coming months, I agree with Mitch's lighter touch here. I think it's reasonable to continue displaying all of this context:
* we're forced to revisit aspects of BFD's authentication strategy in PI5 with certificate expiration later this year
* we're starting to take a closer look at the documentation strategy in the BFD project